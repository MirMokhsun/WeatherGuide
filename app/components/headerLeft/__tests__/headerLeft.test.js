import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HeaderLeft from '..';

describe('HeaderLeft', () => {
    test('HeaderLeft snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<HeaderLeft />);
        expect(snapshot).toMatchSnapshot();
    });
});