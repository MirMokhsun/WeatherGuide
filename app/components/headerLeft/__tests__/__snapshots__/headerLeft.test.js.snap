// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLeft HeaderLeft snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "alignItems": "flex-start",
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "marginTop": 4,
    }
  }
>
  <Icon
    allowFontScaling={false}
    color="rgba(0,0,0,0.7)"
    name="menu"
    size={40}
    style={
      Object {
        "marginHorizontal": 5,
      }
    }
  />
</TouchableOpacity>
`;
